javascript/examples/rollup/rollup.config.js

```diff
@@ -0,0 +1,100 @@
+import alias from "@rollup/plugin-alias"
+import nodeResolve from "@rollup/plugin-node-resolve"
+import replace from "@rollup/plugin-replace"
+import wasm from "@rollup/plugin-wasm"
+import { defineConfig } from "rollup"
+import tla from "rollup-plugin-tla"
+
+const inPageConfig = {
+  input: "src/index.js",
+  output: {
+    file: "public/main.js",
+  },
+  // Fixes the `this` instance at top level in automerge's `stable.js` file
+  context: "globalThis",
```

Comment:
This _might_ be something that could be fixed in `automerge`'s published package